ci-operator/step-registry/ipi/conf/gcp/custom-endpoints/ipi-conf-gcp-custom-endpoints-commands.sh

```diff
@@ -73,6 +73,12 @@ if [[ "${ENABLE_CUSTOM_ENDPOINT_FOR_STORAGE}" == "yes" ]]; then
 EOF
 fi
 
+# The region is not set, because the private service connect endpoint is a global configuration.
+cat >> "${PATCH}" << EOF
+    endpoint:
+      - name: ${gcp_custom_endpoint}
```

Comment:
nit: As this is not conditional, it would make sense to add it in the initial patch starting around line 14, but this is fine as is.